gen3utils/main.py

```diff
@@ -48,16 +49,23 @@ def validate_portal_config(
             logger.error("No dictionary URL in manifest {}".format(manifest_file))
             return
 
-    recorded_errors = val_gitops(dictionary_url, etl_mapping_file, gitops_file)
-
-    if recorded_errors and repository and pull_request_number:
-        if not "GITHUB_TOKEN" in os.environ:
-            logger.error("Exiting: Missing GITHUB_TOKEN")
-            sys.exit(1)
+    recorded_errors, ok = val_gitops(
+        dictionary_url, etl_mapping_file, portal_config_file
+    )
 
     if recorded_errors:
         for err in recorded_errors:
             logger.error("  - {}".format(err))
+        if ok and repository and pull_request_number:
+            if not "GITHUB_TOKEN" in os.environ:
+                logger.error("Exiting: Missing GITHUB_TOKEN")
+                sys.exit(1)
+        comment_gitops_errors_on_pr(repository, pull_request_number, recorded_errors)
+
+    if not ok:
```

Comment:
should be "if not ok or recorded_errors". If there are ETL-portal validation errors, the travis check should fail even if the dictionary-portal validation succeeded.

I would suggest either:
- renaming this "ok" variable to something like "passed_dictionary_validation" AND "recorded_errors" to "etl_mapping_validation_errors" so the code is easier to understand,
- OR changing the logic so that "ok" means both ETL-portal validation AND dictionary-portal validation succeeded